index.js

```diff
@@ -272,22 +272,37 @@ function startSyncWithRetry(watcherArray, maximumNumberOfAttempts, pauseSeconds)
 }
 
 app.listen(config.net.port, () => {
-  // delete existing index and create new index with spatial mapping
 
+  // delete existing index and create new index with spatial mapping
   esclient.indices.exists({ index: config.elasticsearch.index })
   .then(function (resp) {
-    if (resp) {
+    // delete possibly existing index if deleteIndexOnStartup is true
+    if (resp && config.elasticsearch.deleteIndexOnStartup) {
       debug('Index %s already exists: %s', config.elasticsearch.index, resp);
```

Comment:
Adjust log statement to mention if it will be deleted or not. 

Add another log statement for result of deletion (i.e. don't directly `return esclient...delete`).